src/main/kotlin/org/malachite/estella/mails/Payloads.kt

```diff
@@ -52,10 +52,11 @@ fun Interview.toInterviewHostInvitationAsMailPayload(offer: Offer, application:
         else -> ""
     }
     val url = "${MAIN_URL}interview/$role${this.id}/${offer.creator.organization.id}"
+    val password = if (role == "technical/") this.applicationStage.tasksStage!!.id.toString() else ""
```

Comment:
yyy nie wiem, czy to dobry pomysł. Wtedy przy hrowym będzie "your password is:" i będzie pusty string. Lepiej raczej już całe to zdanie ubrać w zmienną, albo przynajmniej dać nullowalne hasło i tam zrobić ifa na to zdanie całe.

Reply:
zstąp głebiej XD w sumie jak to napisałes to dostrzegam że nie jest to zbyt szczęśliwe podejście, ale oifowałem to w MailTextsach, że jak jest hasło to jest cały fragmencik o "Your password is....", a jak hasło jest puste to nic nie ma.

Reply:
Dobra, faktycznie, nie zauważyłem tych ifów w tekście maila. Wyniósłbym to wyżej, bo w tym tekście to tak niefortunnie to ułożone. W sensie nawet zmienną w tamtej funkcji zrobił tak, żeby ten if był poza tekstem